### PR TITLE
Clean repo and avoid duplicating the ipfs website 

### PR DESCRIPTION
**Pull request by TheDiscordian**
As discussed with @2color, removed most of the content of the repository as the information lives elsewhere. The only impacting change here I think is moving the paper which was contained in the repository, which could break some links.

### CODE REVIEW

File: ipfs-p2p-file-system.pdf
Comment (2color) [inline ipfs-p2p-file-system.pdf:1]: I searched google and could only find [one backlink](https://www.google.com/search?q=%22https%3A%2F%2Fgithub.com%2Fipfs%2Fipfs%2Fblob%2Fmaster%2Fpapers%2Fipfs-cap2pfs%2Fipfs-p2p-file-system.pdf%22&rlz=1C5CHFA_enDE1007DE1010&ei=n1ITZOmmNsONkwXhiomgBA&ved=0ahUKEwip8oC8_eD9AhXDxqQKHWFFAkQQ4dUDCBA&uact=5&oq=%22https%3A%2F%2Fgithub.com%2Fipfs%2Fipfs%2Fblob%2Fmaster%2Fpapers%2Fipfs-cap2pfs%2Fipfs-p2p-file-system.pdf%22&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQA0oECEEYAVC1B1j7CWDlDGgBcAB4AIABX4gBtgGSAQEymAEAoAEBwAEB&sclient=gws-wiz-serp) to this file. Whoever looks for it, shouldn't have too much trouble finding the paper :).

File: README.md [hunk 1]
```diff
@@ -1,186 +1,9 @@
...
-
-### Additional resources
-
-- [distributions](https://github.com/ipfs/distributions) - Source code for the IPFS distributions website, https://dist.ipfs.tech.
-- [infra](https://github.com/ipfs/infra) - Tools for maintaining infrastructure for the IPFS community.
-- [testground](https://github.com/ipfs/testground) - Tools for testing distributed software at scale.
-- [ipfs-cluster](https://github.com/ipfs/ipfs-cluster) - Provides data orchestration across a swarm of IPFS daemons by allocating, replicating, and tracking a global pinset distributed among multiple peers.
-- [ipfs-shipyard](https://github.com/ipfs-shipyard) - A wide range of incubated projects by and for the IPFS community.
-- [website](https://github.com/ipfs/ipfs-website) - Source code for the IPFS website, http://ipfs.tech.
+For papers on IPFS, please see the ["Further Reading"](https://docs.ipfs.tech/concepts/further-reading/academic-papers) section of the [IPFS docs site](https://docs.ipfs.tech).
```
Comment (2color) [inline README.md:5]:
```suggestion
For papers on IPFS, please see the [**Academic Papers**](https://docs.ipfs.tech/concepts/further-reading/academic-papers) section of the [IPFS Docs](https://docs.ipfs.tech).
```

**Review by 2color — APPROVED** [review]
Nice! Thanks for cleaning this

File: ipfs-p2p-file-system.pdf
Comment (brodybits) [reply ipfs-p2p-file-system.pdf:1]: Why not overwrite `papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf` with a single-page PDF that tells a user where to look for the updated info?

**Pull request merged**